minigalaxy/installer.py

```diff
@@ -141,3 +148,34 @@ def remove_installer(installer):
 
 def uninstall_game(game):
     shutil.rmtree(game.install_dir, ignore_errors=True)
+
+# For each installation of Windows game, download the latest version fo DXVK and install it.
+def install_dxvk(game):
+    ## Better approach possible with request and regex - Not so familiar :/
+    curl_command = 'curl -s https://github.com/doitsujin/dxvk/releases/latest 2>&1 | grep -Po [0-9]+\.[0-9]+\.[0-9]+'
+    version = subprocess.check_output(curl_command, shell=True)
+    version = version.decode("utf-8").strip()
+
+    prefix_dir = os.path.join(game.install_dir, "prefix")
+    dxvk_dir = os.path.join(CACHE_DIR, "DXVK")
+    dxvk_path = os.path.join(dxvk_dir, 'dxvk-{}.tar.gz'.format(version))
+
+    link = 'https://github.com/doitsujin/dxvk/releases/download/v{}/dxvk-{}.tar.gz'.format(version, version)
+    dxvk_script = os.path.join(dxvk_dir, "dxvk-{}".format(version))
+
+    if not os.path.exists(dxvk_dir):
+        os.makedirs(dxvk_dir, mode=0o755)
+
+    if not os.path.exists(dxvk_script):
+        download = Download(link, dxvk_path)
```

Comment:
Archive extraction and installation should  be a finish_func of Download object.